app/src/main/kotlin/com/wire/android/ui/calling/OngoingCallViewModel.kt

```diff
@@ -42,7 +46,20 @@ class OngoingCallViewModel @Inject constructor(
     }
 
     private suspend fun initiateCall() {
-            //TODO pass conversation type
-            startCall.invoke(conversationId)
+        //TODO pass conversation type
+        startCall.invoke(conversationId)
+    }
+
+    fun hangUpCall() {
+        viewModelScope.launch {
+            endCall.invoke(conversationId)
```

Comment:
small thingy but to keep it consistent I think we do not use the .invoke() but just use the use case as this endCall(converstaionId)